.github/workflows/release.yml

```diff
@@ -16,11 +16,14 @@ jobs:
           node-version: "20.x"
           # registry-url is required for releasing packages
           registry-url: "https://registry.npmjs.org"
+
+      - name: Install latest npm cli
+        run: npm install -g npm@latest
```

Comment:
Why do we need this?

Reply:
> As trusted publishing requires npm CLI version >=11.5.1, we manually install the latest version since the default installed npm version is insufficient. 

In the [latest workflow run](https://github.com/SAP/openapi-specification/actions/runs/17095131883/job/48477682672#step:3:12) we can see that by default the npm cli version `10.8.2` is used, which does not support trusted publishing.